config/environments/production.rb

```diff
@@ -5,9 +5,9 @@
   config.cache_classes = true
   config.eager_load = true
   config.consider_all_requests_local = false
-  config.action_controller.asset_host = Figaro.env.domain_name
   config.action_controller.perform_caching = true
-  config.serve_static_files = ENV['RAILS_SERVE_STATIC_FILES'].present?
+
+  config.action_controller.asset_host = Figaro.env.asset_host
```

Comment:
Do we want to have a default in case this isn't set?

Reply:
Good point. We should either make it required, or fall back to serving assets from nginx.